webservices/resources/legal.py

```diff
@@ -115,11 +115,6 @@ def get(self, q="", from_hit=0, hits_returned=20, **kwargs):
         else:
             doc_types = [kwargs.get("type")]
 
-            # if doc_types is not in one of ALL_DOCUMENT_TYPES
-            # then reset type = all (= ALL_DOCUMENT_TYPES)
```

Comment:
These lines are not needed because we verify that doc_type is correct in args.py